charts/skypilot/values.yaml

```diff
@@ -222,6 +222,9 @@ gcpCredentials:
   # TODO(romilb): This can be made optional by using the project in the key json by default.
   projectId: null
 
+# Populate RunPod credentials from a secret `runpod-credentials` with key `api_key`
+runpodCredentials:
+  enabled: false
```

Comment:
Should we make the secret name configurable?

Reply:
I tried to follow existing convention as much as possible. I do think it makes sense to make the secret name configurable, but then we should do the same with AWS and GCP, and (for that matter), should we make the secret keys configurable too?

Reply:
(In which case, that should probably be a separate PR)

Reply:
Sounds good!